src/tokens/shasta.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
```

Comment:
this is the zero address for tron `T9yD14Nj9j7xAB4dbGeiX9h8unkKHxuWwb`

Reply:
Done!